src/test/java/io/strimzi/kafka/quotas/TickingClock.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.quotas;
+
+import java.time.Clock;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+
+public class TickingClock extends Clock {
```

Comment:
We should rename this as it doesn't actually `Tick` by itself. Instead it ticks on demand. So `TickableClock`?